Update README.md (#19938)
DataDog/integrations-core: sqlserver/README.md

     - host: "<SQL_HOST>,<SQL_PORT>"
       username: datadog
       password: "<YOUR_PASSWORD>"
       connector: adodbapi 

>        adoprovider: MSOLEDBSQL19  # Replace with MSOLEDBSQL for versions 18 and lower

Reviewer: ```suggestion
       adoprovider: MSOLEDBSQL19  # Replace with MSOLEDBSQL for versions 18 and previous
```

   ```

    If you use port autodiscovery, use `0` for `SQL_PORT`. See the [example check configuration][6] for a comprehensive description of all options, including how to use custom queries to create your own metrics.
    

>     Use [supported drivers] (https://docs.datadoghq.com/database_monitoring/setup_sql_server/selfhosted/?tab=sqlserver2014#supported-drivers) based on your SQL Server setup.

Reviewer: ```suggestion
    Use [supported drivers][25] based on your SQL Server setup.
```

[21]: https://www.datadoghq.com/blog/sql-server-performance
[22]: https://www.datadoghq.com/blog/sql-server-metrics
[23]: https://www.datadoghq.com/blog/migrate-sql-workloads-to-azure-with-datadog/

> [24]: https://www.datadoghq.com/blog/optimize-sql-server-performance-with-datadog/

Reviewer: ```suggestion
[24]: https://www.datadoghq.com/blog/optimize-sql-server-performance-with-datadog/
[25]: https://docs.datadoghq.com/database_monitoring/setup_sql_server/selfhosted/#supported-drivers
```
